recipes/_install_rbenv.rb

```diff
@@ -1,10 +1,14 @@
 include_recipe 'ruby_rbenv::default'
-include_recipe 'ruby_rbenv::system_install'
 include_recipe 'ruby_build::default'
 
-rbenv_ruby node['samson']['ruby']['version'] do
-  user node['samson']['user']
-end
+node.default['rbenv']['user_installs'] = [
```

Comment:
Yeah the issue here is that since attribute files are processed in alphabetical order, if I follow our naming convention then the rbenv attribute file would be read before the samson one, making it impossible to lookup the proper values for the samson user to build out this one.

So either we name the attribute file in a funky way to make it be parsed after samson, we rename the samson attribute file to default to make sure its parsed first, we leave the attribute change in the recipe or we define the values for the install of rbenv separately from the samson attributes, which would mean that there are 2 sets of attributes to change to use a different user for samson.